orbit/pkg/token/reader_test.go

```diff
@@ -0,0 +1,124 @@
+package token
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func setup(t *testing.T) *os.File {
+	t.Helper()
+	file, err := os.CreateTemp("", "")
```

Comment:
You can simplify by using [TempDir](https://pkg.go.dev/testing#T.TempDir)

Reply:
what I like about this is that I get a `*File` instance to call methods like `Stat`, `ModTime`, etc